where-to-travel/src/main/webapp/script.js

```diff
@@ -199,8 +199,11 @@ function getLocationFromUserInput() {
  */
 function addPlacesFromDirection(lat, lng, place_candidates) {
   return new Promise(function(resolve) {
-    const sw = new google.maps.LatLng(lat - .4, lng - 4);
-    const ne = new google.maps.LatLng(lat + .4, lng + 4); 
+    const halfWidth = 0.5;
```

Comment:
Add variables for specifying width and height of bounding box instead of directly adding to passed in latitude and longitude. Makes it more clear what the code is actually doing.